docs/user/BuildWithPrebuilt.md

```diff
@@ -0,0 +1,28 @@
+# Build from sources using Prebuilt Binaries
```

Comment:
This looks to be a copy of the text from the OR docs. We should always avoid having the same instructions in two different places as this makes it more difficult to keep up-to-date. Please link to OR-docs instead. Also, here we will need to add instructions on how to install Yosys and Klayout using `apt`/`.deb` files. Please also double-check if the user needs to set any special PATH variable to run `make` commands after installing the prebuilt binaries.

Reply:
@vvbandeira I think the concern was that we wanted to separate the prebuilt binaries install instructions from built local instructions. Should I modify the OR docs as well?

Reply:
(I think we should remove the `Build from sources` as it does not make much sense with prebuilt bins)

Adding the new file is correct, but we do not want to duplicate the instructions from OR docs here. This new file (i.e., `docs/user/BuildWithPrebuilt.md`) should have sections similar to the following:

```
# Using Prebuilt Binaries

## Install Yosys and Klayout

Either: apt install -y yosys klayout
Or the necessary steps to download and install, will need to double-check their docs.

## Install OpenROAD

[link to OR docs]

## Aditional setup

> might be necessary, not sure
```

Reply:
Ping me on slack if a quick meeting would be better to understand.

Reply:
Agree with Vitor-Remove Build from sources for pre-built. also change the spelling to Pre-built not prebuilt